src/sentry/signals.py

```diff
@@ -53,6 +53,10 @@ def send_robust(self, sender, **named):
 pending_delete = BetterSignal(providing_args=["instance", "actor"])
 event_processed = BetterSignal(providing_args=["project", "event"])
 
+# This signal should eventually be removed as we should not send
+# transactions through post processing
+transaction_processed = BetterSignal(providing_args=["project", "event"])
```

Comment:
Won't you need this to keep receivers/transactions working?

Reply:
Yes, I'm planning to incrementally move that functionality into save_event or remove if we can live without it. Hopefully transaction_processed can eventually go away, and we won't need to send transactions through post processing at all.